bigtable/export_test.go

```diff
@@ -71,6 +77,7 @@ type IntegrationEnv interface {
 	NewInstanceAdminClient() (*InstanceAdminClient, error)
 	NewClient() (*Client, error)
 	Close()
+	GetPeer() *peer.Peer
```

Comment:
This can just be called `Peer`. See https://golang.org/doc/effective_go.html#Getters.

Reply:
Fixed. Thanks!